### DISCUSSION TITLE
Complete the expression

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1127] [Galaxy Trivia] [answered]
> Go to 😇 for the ... and 😈 for the ...

**Bonus:** Name the author.

**Comment by chinchern** [comment]
Go to 😇 for the climate and 😈 for the company

by Mark Twain

**Comment by peterbaccaro** [comment, marked as answer]
The full expression is **"Go to Heaven for the climate, Hell for the company."** and the expression is attributed to **Mark Twain**.
